tests/addresslib/parser_mailbox_test.py

```diff
@@ -540,10 +531,6 @@ def test_local_part():
         EmailAddress(sample_qtext, '"{0}"@b'.format(sample_qtext)))
 
     # fail qtext
-    run_mailbox_test('"{0}""{0}"@b'.format(sample_qtext), None)
-    run_mailbox_test('"john""smith"@b'.format(sample_qtext), None)
-    run_mailbox_test('"{0}" @b'.format(sample_qtext), None)
```

Comment:
Valid addresses according to the RFC. Optional whitespace is allowed around quoted strings.